apps/api/src/app/integrations/usecases/update-integration/update-integration.usecase.ts

```diff
@@ -32,6 +32,22 @@ export class UpdateIntegration {
       throw new NotFoundException(`Entity with id ${command.integrationId} not found`);
     }
 
+    const identifierHasChanged = command.identifier && command.identifier !== existingIntegration.identifier;
+    if (identifierHasChanged) {
+      const existingIntegrationWithIdentifier = await this.integrationRepository.findOne({
+        _organizationId: command.organizationId,
+        identifier: command.identifier,
+      });
+
+      if (existingIntegrationWithIdentifier) {
+        throw new BadRequestException('Integration with identifier already exists');
+      }
+    }
```

Comment:
update request: when identifier has changed, check if we don't have another similar in the DB, and if we have throw an error